js/src/css/shared/_woocommerce-admin.scss

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
Most of the CSS was moved from `<FullScreen>` and `<FullContainer>` and merged to a single file. It'd be easier to check the diff from [this link](https://github.com/woocommerce/google-listings-and-ads/pull/1206/files/89cf9aa6d8434cef64ef30146a5b0c3970510ea6..a5e8509020200139384ed11f48302db2d87a1e5a).